geoh5py/objects/surveys/electromagnetics/base.py

```diff
@@ -173,6 +177,27 @@ def add_validate_component_data(self, name: str, data_block: list | dict):
 
         return prop_group
 
+    def base_copy(
+        self,
+        parent: Group | Workspace | None = None,
+        copy_children: bool = True,
+        clear_cache: bool = False,
+        mask: np.ndarray | None = None,
+        **kwargs,
+    ):
+        """
+        Call the super().copy of the class.
```

Comment:
**suggestion (doc):** explain the difference between `base_copy()` and `copy()`, or when to call one rather than the other.

Or maybe `base_copy()` can be renamed to convey a better clue on its role.

Reply:
How about we make a private method `_super_copy`. Should only be used as internal mechanics of the class.